src/vm_interrupts.rs

```diff
@@ -24,6 +24,7 @@ pub type Result<T> = core::result::Result<T, Error>;
 enum BindStatus {
     Binding(CpuId, ImsicFileId),
     Bound(CpuId, ImsicFileId),
+    Rebinding(CpuId, ImsicFileId, CpuId, ImsicFileId),
```

Comment:
Pre-clone and post-clone `Rebinding` should be separate states so that we can enforce that the host does the clone step.

Reply:
Fixed.